api/src/db/models/opportunity_models.py

```diff
@@ -451,3 +457,80 @@ class OpportunityChangeAudit(ApiSchemaTable, TimestampMixin):
         BigInteger, ForeignKey(Opportunity.opportunity_id), primary_key=True, index=True
     )
     opportunity: Mapped[Opportunity] = relationship(Opportunity)
+
+
+class OpportunityCompetition(ApiSchemaTable, TimestampMixin):
+    __tablename__ = "opportunity_competition"
+
+    competition_id: Mapped[uuid.UUID] = mapped_column(UUID, primary_key=True, default=uuid.uuid4)
+    opportunity_id: Mapped[int] = mapped_column(
+        BigInteger, ForeignKey(Opportunity.opportunity_id), index=True
+    )
+    opportunity: Mapped[Opportunity] = relationship(Opportunity)
+
+    legacy_competition_id: Mapped[int | None] = mapped_column(BigInteger, index=True)
+    public_competition_id: Mapped[str | None]
+    legacy_package_id: Mapped[str | None]
+    competition_title: Mapped[str | None]
+
+    opening_date: Mapped[date | None]
+    closing_date: Mapped[date | None]
+    grace_period: Mapped[int | None] = mapped_column(BigInteger)
+    contact_info: Mapped[str | None]
+
+
+class OpportunityCompetitionInstruction(ApiSchemaTable, TimestampMixin):
+    __tablename__ = "opportunity_competition_instruction"
+
+    competition_instruction_id: Mapped[uuid.UUID] = mapped_column(
+        UUID, primary_key=True, default=uuid.uuid4
+    )
+    competition_id: Mapped[uuid.UUID] = mapped_column(
+        UUID, ForeignKey(OpportunityCompetition.competition_id), index=True, primary_key=True
+    )
+    opportunity_competition: Mapped[OpportunityCompetition] = relationship(OpportunityCompetition)
+
+    file_location: Mapped[str]
+
+
+class OpportunityCompetitionAssistanceListing(ApiSchemaTable, TimestampMixin):
+    __tablename__ = "opportunity_competition_assistance_listing"
+
+    competition_id: Mapped[uuid.UUID] = mapped_column(
+        UUID, ForeignKey(OpportunityCompetition.competition_id), index=True, primary_key=True
+    )
+    opportunity_competition: Mapped[OpportunityCompetition] = relationship(OpportunityCompetition)
+
+    opportunity_assistance_listing_id: Mapped[int] = mapped_column(
+        BigInteger,
+        ForeignKey(OpportunityAssistanceListing.opportunity_assistance_listing_id),
+        primary_key=True,
+    )
+    opportunity_assistance_listing: Mapped[OpportunityAssistanceListing] = relationship(
+        OpportunityAssistanceListing
+    )
+
+
+class OpportunityApplicationForm(ApiSchemaTable, TimestampMixin):
+    __tablename__ = "opportunity_application_form"
+
+    form_id: Mapped[uuid.UUID] = mapped_column(UUID, primary_key=True, default=uuid.uuid4)
+    form_name: Mapped[str]
+    form_version: Mapped[str]
+    is_active: Mapped[bool]
+    description: Mapped[str]
```

Comment:
Yknow what, on second thought, I don't even know what this description refers to, let's drop this column. It's not on the form UI, so let's hold off on it for now